snorkel/labeling/model/label_model.py

```diff
@@ -50,10 +50,6 @@ def _create_L_ind(self, L):
 
         Note that no column is required for 0 (abstain) labels.
         """
-        # TODO: Update LabelModel to keep L variants as sparse matrices
-        # throughout and remove this line.
-        if issparse(L):
```

Comment:
So we're requiring a manual cast now?

Reply:
The LF appliers spit out a CSR. Should we change that?

Reply:
this was a repeat of line 32 in the same file (`_check_L`) which already converts it to a dense matrix if it's sparse